docs/rackhd/graph_progress.rst

```diff
@@ -0,0 +1,146 @@
+Workflows Progress
```

Comment:
I suggest changing it to "workflows progress notification" to better align with the feature goal.